visualization/package.json

```diff
@@ -1,174 +1,174 @@
-		"jquery": "^3.3.1",
-		"klona": "^2.0.4",
-		"lodash.camelcase": "^4.3.0",
-		"lodash.debounce": "^4.0.8",
-		"lodash.isequal": "^4.5.0",
-		"lodash.throttle": "^4.1.1",
-		"md5": "^2.3.0",
-		"nouislider": "^14.6.0",
-		"nw": "^0.50.0",
-		"open": "^7.3.0",
-		"redux": "^4.0.4",
-		"shelljs": "^0.8.1",
-		"three": "^0.124.0",
-		"three-orbit-controls": "^82.1.0",
-		"typescript-json-schema": "^0.47.0"
-	},
-	"devDependencies": {
-		"@babel/cli": "^7.10.4",
-		"@babel/core": "^7.10.4",
-		"@babel/preset-env": "^7.10.4",
-		"@babel/preset-typescript": "^7.10.4",
-		"@types/angular": "^1.7.2",
-		"@types/angular-mocks": "^1.5.11",
-		"@types/color-convert": "^2.0.0",
-		"@types/d3-hierarchy": "^2.0.0",
-		"@types/expect-puppeteer": "^4.4.3",
-		"@types/jest": "^26.0.4",
-		"@types/jest-environment-puppeteer": "^4.4.0",
-		"@types/lodash": "^4.14.123",
-		"@types/node": "^14.0.14",
-		"@types/puppeteer": "^5.4.0",
-		"@typescript-eslint/eslint-plugin": "^4.0.1",
-		"@typescript-eslint/parser": "^4.0.1",
-		"angularjs-color-picker": "^3.4.8",
-		"babel-loader": "^8.1.0",
-		"bestzip": "^2.1.5",
-		"clean-webpack-plugin": "^3.0.0",
-		"css-loader": "^5.0.0",
-		"es6-shim": "^0.35.3",
-		"eslint": "^7.11.0",
-		"eslint-config-prettier": "^7.0.0",
-		"eslint-plugin-unicorn": "^24.0.0",
-		"file-loader": "^6.0.0",
-		"html-loader": "^1.1.0",
-		"html-loader-jest": "^0.2.1",
-		"html-webpack-plugin": "^4.3.0",
-		"jest": "^26.1.0",
-		"jest-puppeteer": "^4.4.0",
-		"jest-serializer-html": "^7.0.0",
-		"lint-staged": "^10.5.2",
-		"makeshift": "^1.1.0",
-		"node-sass": "^5.0.0",
-		"nwjs-builder-phoenix": "^1.15.0",
-		"plop": "^2.7.3",
-		"puppeteer": "^5.4.1",
-		"rimraf": "^3.0.2",
-		"sass-loader": "^10.0.1",
-		"style-loader": "^2.0.0",
-		"ts-jest": "^26.1.1",
-		"ts-loader": "^8.0.3",
-		"typescript": "^4.0.2",
-		"webpack": "^4.41.2",
-		"webpack-cli": "^4.1.0",
-		"webpack-dev-server": "^3.11.0",
-		"webpack-glsl-loader": "^1.0.1"
-	},
-	"optionalDependencies": {
-		"7zip-bin-linux": "^1.2.0",
-		"7zip-bin-mac": "^1.0.1",
-		"7zip-bin-win": "^2.1.1"
-	}
-}
+{
+	"author": "MaibornWolff GmbH <dist_intern_codecharta@maibornwolff.de>",
+	"homepage": "https://maibornwolff.github.io/codecharta/",
+	"bugs": {
+		"url": "https://github.com/MaibornWolff/codecharta/issues",
+		"email": "dist_intern_codecharta@maibornwolff.de"
+	},
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/MaibornWolff/codecharta.git"
+	},
+	"name": "codecharta-visualization",
+	"version": "1.67.0",
+	"description": "MaibornWolff CodeCharta",
+	"files": [
+		"dist/webpack",
+		"app",
+		"conf",
+		"Dockerfile",
+		"LICENSE.md",
+		"package.json",
+		"package-lock.json",
+		"README.md",
+		"tsconfig.json",
+		"cli.js"
+	],
+	"codecharta": {
+		"apiVersion": "1.2"
+	},
+	"window": {
+		"icon": "app/icon.png",
+		"toolbar": true,
+		"frame": true,
+		"width": 1024,
+		"height": 768
+	},
+	"main": "dist/webpack/index.html",
+	"build": {
+		"nwVersion": "0.49.0",
```

Comment:
We might want to update this as well :-)

Reply:
Oh, you are right
I'm not sure if that is what causes the issue in the first place now
That would explain why there are no mentiones about this on their official github page?

Reply:
So we just have too much of a version difference between the build and the used package and thats what makes it fail, could that be possible?

Reply:
@BridgeAR wdyt?

Reply:
Ok setting both to 0.51 did not solve the issue, I guess we need to roll back then